scala3doc/src/dotty/dokka/tasty/comments/MemberLookup.scala

```diff
@@ -59,7 +59,7 @@ trait MemberLookup {
     import dotty.tools.dotc
     given dotc.core.Contexts.Context = rootContext.asInstanceOf
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
-    val members = sym.info.decls.iterator
+    val members = sym.info.decls.iterator.filter(_.denot.isAbsent(false))
```

Comment:
Should be `isAbsent(canForce = true)` if you don't want false positives.

Reply:
Interestingly enough `canForce = true` results in exceptions, but `canForce = false` doesn't result in false positives in cases we care about.